bin/reth/src/config.rs

```diff
@@ -78,3 +80,12 @@ impl Default for SenderRecoveryConfig {
         Self { commit_threshold: 5_000, batch_size: 1000 }
     }
 }
+
+/// Configuration for each stage in the pipeline.
+#[derive(Debug, Clone, Default, Deserialize, Serialize)]
+pub struct PeersConfig {
+    /// Trusted nodes to connect to.
+    pub trusted_nodes: Vec<String>,
```

Comment:
should be NodeRecord once we have Serialize/Deserialize
```suggestion
    pub trusted_nodes: Vec<NodeRecord>,
```

Reply:
Changed